public/index.php

```diff
@@ -3,38 +3,29 @@
 declare(strict_types=1);
```

Comment:
This was removed from the latest Symfony

Reply:
Good practice is to leave it on, at least for projects.

Symfony removed it for a simple reason, made it easier to maintain the Symfony framework for versions lower than php7.

https://github.com/symfony/recipes/pull/507#discussion_r259587271
https://github.com/symfony/symfony/issues/28423#issuecomment-466927534